src/Utils/functions.ts

```diff
@@ -0,0 +1,107 @@
+import { constraints } from "../constants";
```

Comment:
lets discuss on all these functions in our daily sync

Reply:
naming needs to be improved for functions

Reply:
changes the function names